Server/VikingVault.API/Controllers/LoginController.cs

```diff
@@ -0,0 +1,33 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using System.Threading.Tasks;
+using VikingVault.Services.Abstractions;
+using VikingVault.DataAccess.Models;
+
+namespace WebApi.Controllers
+{
+    [Authorize]
+    [ApiController]
+    [Route("api/[controller]")]
+    public class LoginController : ControllerBase
+    {
+        private ILoginService _loginService;
+
+        public LoginController(ILoginService loginService)
+        {
+            _loginService = loginService;
+        }
+
+        [AllowAnonymous]
+        [HttpPost("authenticate")]
+        public IActionResult Authenticate([FromBody]UserLogin userParam)
+        {
+            var user = _loginService.Authenticate(userParam.Email, userParam.Password);
+
+            if (user == null)
```

Comment:
It's better to not use single line if/else statement, when you always add curly braces to those kind of statement it makes it easier to read and also might prevent some errors. Also, it's better to have a consistent coding style.